basic/src/test/java/org/jboss/hal/testsuite/test/runtime/SimpleWebserviceEndpointTestCase.java

```diff
@@ -159,14 +155,17 @@ public void callServiceAndGetMetrics() throws Exception {
 
         wsePage.navigateInDeploymentsMenu();
```

Comment:
Is it possible to 
1. move `WebServiceEndpointsPage.navigateInDeploymentsMenu()` to `DeploymentPage` ?
2. merge `DeploymentPage` with `DomainDeploymentPage` ?
Might be in another pull request.